### ISSUE TITLE
suggestion: multi pc stream setup

### ISSUE THREAD

**Issue by koin612** [dethredic/WC3StreamerOverlay#8]
Hi,

some streamers are using 2 pc setup (one for gaming one for streaming).
currently websocket server only allows localhost connections. 

I recently made a quickfix for a friend, but it's not PR ready, because I can't figure out a user friendly solution.

maybe server sends udp broadcasts with its ipv4, client listens and sets it?
might need an external tool for this, or can node stuff be webpacked? any other ideas?

**Comment by dethredic** [comment]
Can you create a config file or env variable with the server IP, then have both the python server and JS client read that and use the specified IP?

**Comment by koin612** [comment]
yes I thought about this, too. But I don't think the released JS client will be able to read stuff from the filesystem, because it would violate the Browser sandbox design?

**Comment by dethredic** [comment]
I'm not sure off the top of my head, but I'll look into it. I'm not a web developer by any means.

**Comment by suuN91** [comment]
Hey first thanks for your awesome work! 
Just wanted to ask if you ever implemented a way for multi pc-streaming! or if you are still planing  it !

**Comment by koin612** [comment]
i think if we wanna make the client configurable to enable multi pc streaming, we have to serve it from the backend kinda like a webserver

**Comment by suuN91** [comment]
you made a "quickfix" for a friend. how is that working ?

**Comment by JFGHT** [comment]
I don't understand your "read stuff from the filesystem" comment, @koin612.

You are not reading data from files, you are serving it with websockets. Why wouldn't you be able to put the local IP+port of the playing computer at the websocket connection (in the client/streaming computer) and make it work?

**Comment by koin612** [comment]
I was thinking about a elegant way to tell the client the ip and port of the websocket server (for normal users). @JFGH

But I guess including a js file where the data could be edited should be easy enough for both sides
